### DISCUSSION TITLE
Lock is missing option for Open on Lovelace more info panel

### DISCUSSION THREAD

**Discussion by bkbilly** [home-assistant/frontend#13013] [Dashboard] [answered]
I have created a lock using ESPHome which supports Lock/Unlock/Open options. 
I am able to run all these services using the lock.lock, lock.unlock, lock.open, but they don't seem to exist on the frontend as you can see from this screenshot:
![image](https://user-images.githubusercontent.com/518494/175822854-145c6edc-dc1e-4e44-8e42-337c2c771d5a.png)

**Comment by zsarnett** [comment]
Lock and Unlock are the only options that are available via the frontend. What does lock.open even do?

**Comment by bkbilly** [comment]
The open is used for opening the latch of the door where there is no handle from the outside.

**Comment by KTibow** [comment]
This would require checking supported-features and adding a button to [more-info-lock.ts](https://github.com/home-assistant/frontend/blob/dev/src/dialogs/more-info/controls/more-info-lock.ts).

**Comment by bkbilly** [comment, marked as answer]
I forgot that I created this discussion, so I created another here: https://github.com/home-assistant/frontend/discussions/14821
Also I managed to fix it myself in this pull request: https://github.com/home-assistant/frontend/pull/14852
